tsconfig.rollup.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
does it make sense to embed this config directly in the `rollup.config.js`? I feel like otherwise it's just another file added to the heap of tooling files laying around. Is there a benefit to having this in its own file that I'm missing?

Reply:
Yep sounds good!